pkg/network/encoding/encoding.go

```diff
@@ -87,6 +90,13 @@ func InitConnectionsModeler(conns *network.Connections) *ConnectionsModeler {
 	}
 }
 
+// CleanConnectionModeler cleans all encoders resources.
+func (c *ConnectionsModeler) CleanConnectionModeler() {
```

Comment:
nit: This function is on `ConnectionsModeler` so that part of the name seems superfluous. I'd suggest `Close`.